src/conversions/mod.rs

```diff
@@ -7,3 +7,4 @@ pub mod num_bigint;
 pub mod num_complex;
 mod osstr;
 mod path;
+pub mod serde;
```

Comment:
👍 it doesn't feel _exactly_ like a conversion to me but I do like having it in the same place as all the other optional dependencies!